api_tests/nodes/views/test_node_draft_registration_list.py

```diff
@@ -126,54 +216,37 @@ def test_admin_can_view_draft_list(
         assert data[0]['id'] == draft_registration._id
         assert data[0]['attributes']['registration_metadata'] == {}
 
-    def test_osf_group_with_admin_permissions_can_view(
-            self, app, user, draft_registration, project_public,
-            schema, url_draft_registrations):
-        group_mem = AuthUserFactory()
-        group = OSFGroupFactory(creator=group_mem)
-        project_public.add_osf_group(group, permissions.ADMIN)
-        res = app.get(url_draft_registrations, auth=group_mem.auth, expect_errors=True)
+    def test_node_admin_can_view_draft_list(
```

Comment:
This test is confirming that a Project Admin can view the draft list even if they are not a contributor on the draft -- and, in fact, confirms that they can see the Draft in the list, even if they are not a contributor on the Draft, which was identified as work for a different ticket.